packages/plugin-spatial-nback/src/index.ts

```diff
@@ -0,0 +1,509 @@
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
+
+import { version } from "../package.json";
+
+const info = <const>{
+  name: "@jspsych-contrib/plugin-spatial-nback",
```

Comment:
this can be just "spatial-nback" - I think that's how it is in other plugins. double check.